Atapiave/QA_team
# FIX: Updates the get_otp function to match the new API requirements

## Any background context you want to provide?

The web service that gets the OTP was changed, we need to update the way we get the OTP code since the new web service now returns a JSON.

## What does this PR do?

Adds some headers and a condition to handle the "Enrolamiento Light" phone numbers

## Where should the reviewer start?

From the following file:
* Behave/utils/api_handler.py

## How should this be tested?

By creating an object of the LoginScreen class and using the login method. Or by running the existing Login test as follows:

1.- Clone this branch
2.- Create an .env file by creating a copy of the .env-sample
3.- Fill the .env file with the details of a user that exists in the DEV environment and other required details like:
OTP_API_URL = 'http://public-cluster-fargate-superapp-259682327.us-east-1.elb.amazonaws.com/superapp/enrolamiento/gestion-mensajes/light/v1/obtenerotp?telefono='
4.- Add an android baz-dev.apk inside the Behave/APP/ folder
5.- Open a Terminal window (Could be inside of the IDE you are using)
6.- Get inside the Behave folder
7.- Run the following command:
paver run_feature android login
8.- Check in your physical phone that the login process is being executed.

## What are the relevant tickets?

SUPAUT-1

## Questions :question:
#### Checklist
